src/components/Marketplace/MarketplaceScreen.js

```diff
@@ -1,5 +1,100 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Airtable from 'airtable';
+import {
+  AppBar, IconButton, InputBase, Tab, Tabs, Toolbar,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import './MarketplaceScreen.css';
+import { makeStyles } from '@material-ui/core/styles';
+import FarmCard from './FarmCard';
+
+// Airtable set-up
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+// Custom styling
+const styles = makeStyles({
+  tabBackground: {
+    backgroundColor: 'grey',
+    color: 'green',
+  },
+});
 
 export default function MarketplaceScreen() {
-  return <h1>Marketplace Screen</h1>;
+  const [farmListings, setFarmListings] = useState([]);
+  // Get records from Airtable whenever DOM mounts and updates/changes
+  useEffect(() => {
+    base('Farms').select({ view: 'Grid view' }).all()
+      .then((records) => {
+        setFarmListings(records);
+      });
+  });
+
+  // 3 tab states: 0 (purchase buy), 1 (aggregate buy), 2 (distressed buy)
+  const [currentTab, setCurrentTab] = useState(0);
+
+  return (
+    <div className={styles.tabBackground}>
+      <h1> Marketplace Screen :) </h1>
+      {/* Search bar element */}
+      <div className="searchBarContainer">
```

Comment:
We can remove everything here except for the <h1> Market Screen</h1> and your farmListings mapping! You can store it in another file so we can build off of that afterwards!